components/sections/carousel/carousel.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Marquee from 'react-fast-marquee';
+import { Image } from 'components/shared';
+
+const Carousel = ({ data }) => {
+  const { logos } = data;
+
+  return (
+    <section className="w-full bg-black lg:py-24 py-12" aria-labelledby="carousel-heading">
+      <h2 id="carousel-heading" className="sr-only">
+        Partner Logos Carousel
+      </h2>
+      <Marquee speed={60} gradient={false} pauseOnHover={false} direction="left">
+        {logos.map(logo => (
+          <div key={logo.title} className="py-8 px-10  lg:mx-7 lg:p-10 bg-black">
```

Comment:
For this onboarding, it fine to use the `logo.title` as the `key`.

**Tip:** Logo titles can at times not be unique, then it would be safer to either:
- Use an id value of the image (something like `logo.id` if it exists).
- Combine `logo.title` with `index` value.


```suggestion
        {logos.map((logo, i) => (
          <div key={`${logo.title}-${i}`} className="..."> ... </div>
        ))}

```